main.nf

```diff
@@ -2443,7 +2487,7 @@ process qualimap {
     script:
     def snpcap = params.snpcapture_bed != '' ? "-gff ${params.snpcapture_bed}" : ''
     """
-    qualimap bamqc -bam $bam -nt ${task.cpus} -outdir . -outformat "HTML" ${snpcap}
+    qualimap bamqc -bam $bam -nt ${task.cpus} -outdir . -outformat "HTML" ${snpcap} --java-mem-size=${task.memory.toGiga()}G
```

Comment:
Did that solve your issues with qualimap?
(asking for a friend)

Reply:
Pinging @maxibor who added this - yes, I think so 🤔